portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,40 +26,54 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   static final String COMMENT_INPUT_ID = "user-comment";
+  static final String USERNAME_ID = "user-name";
 
   private List<String> commentsList = new ArrayList<>();
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Gson gson = new Gson();
-    String json = gson.toJson(commentsList);
-
-    // Send the JSON as the response
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
-  }
-
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Get the input from the form.
+    // Get the comment text
     String comment = getComment(request);
-
     if (comment == null) {
       return;
```

Comment:
This might be a bigger topic that gets covered later in your project so it's not a change I'm asking you to figure out right now. However, usually, when a request is bad, an error is somehow surfaced (and results in an error banner or something like that on the UI). You may also want to check for empty strings and not just nulls here.